e3/anod/context.py

```diff
@@ -364,6 +364,25 @@ def add_action(data, connect_with=None):
             if connect_with is not None:
                 self.connect(connect_with, data)
 
+        def add_dep(deps, dep, dep_instance):
+            """Add a new dependency in a dependencies dict.
+
+            :param deps: the Anod instance ``deps`` attribute
+            :type deps: dict
+            :param dep: a Dependency instance
```

Comment:
suggest, for clarity's sake, that this is the dependency we want to add to the dependency list?